docs/source/install/profile.rst

```diff
@@ -207,3 +207,32 @@ Queue: dvd_usr_prod (Nvidia P100 GPUs)
 
 .. literalinclude:: profiles/davide-cineca/gpu_picongpu.profile.example
    :language: bash
+
+JURECA (JSC)
+--------------------
+
+**System overview:** `link <http://www.fz-juelich.de/ias/jsc/EN/Expertise/Supercomputers/JURECA/JURECA_node.html>`_
+
+**User guide:** `link <http://www.fz-juelich.de/ias/jsc/EN/Expertise/Supercomputers/JURECA/UserInfo/UserInfo_node.html>`_
+
+**Production directory:** ``$SCRATCH`` (`link <http://www.fz-juelich.de/SharedDocs/FAQs/IAS/JSC/EN/JUST/FAQ_00_File_systems.html?nn=1297148>`_)
+
+For this profile to work, you need to download the :ref:`PIConGPU source code <install-dependencies-picongpu>` manually.
```

Comment:
You also need PNGwriter, libSplash and ADIOS below. See other profiles above on how to note this here :)

Also, since it's multiple queues please write "For these profiles to work, ..."

Reply:
Good point. Just to clarify a little bit: currently the dependencies installed in a way that for users of my project it is ready out-of-the box. However, ofc generally you are correct and additional dependencies are required. Will fix.